starter-code/src/functions-and-arrays.js

```diff
@@ -11,6 +21,22 @@ var words = [
   'crackpot'
 ];
 
+function findLongestWord (box){
+  var longest = "";
```

Comment:
This function will fail returning undefined when the array is empty. You can solve this checking if the array is empty and returning undefined then. A better way is asign to your comparator (longest) the first array's element. If the array is empty, longest will be 'undefined' and the function will return 'undefined'. Using an internal element for comparision solve some tricky problems (e.g. finding a minimum).
```
function findLongestWord(box) {
  var longest = box[0];

  for (var i = 1; i < box.length; i++) {
    if (box[i].length > longest.length) {
      longest = box[i];
    }
  }
  return longest;
}
```